src/componentprocessor/index.ts

```diff
@@ -4,38 +4,37 @@ import { promisify } from "util";
 import xmldoc, { XmlDocument } from "xmldoc";
 import pSettle = require("p-settle");
 const readFile = promisify(fs.readFile);
-const fg = require("fast-glob");
+import * as fg from "fast-glob";
```

Comment:
what is the difference between import and require?

Reply:
i think import is es6 syntax.  This change was to get rid of linter error :)

Reply:
I think `import` is the "new" es6 syntax and is what the linter expects